core/templates/dev/head/collection_editor/collection_node_editor_directive.html

```diff
@@ -1,35 +1,27 @@
 <script type="text/ng-template" id="inline/collection_node_editor_directive">
-  <div>
-    [<[getCollectionNode().getExplorationId()]>] <em><[getCollectionNode().getExplorationTitle()]></em>
+  <div class="collection-editor-node-title">
+    <[getCollectionNode().getExplorationTitle()]>
+    <span ng-click="deleteCollectionNode()" class="glyphicon glyphicon-remove-circle" aria-hidden="true">
+    </span>
     <span ng-if="!getCollectionNode().doesExplorationExist()"><strong>Warning: This exploration no longer exists.</strong></span>
     <span ng-if="getCollectionNode().isExplorationNew()"><strong>Warning: Please save the collection before details will appear for this exploration.</strong></span>
-    <button ng-click="deleteCollectionNode()">Delete</button>
   </div>
-  <p> Prerequisite skills: </p>
-  <div ng-if="getCollectionNode().getPrerequisiteSkillList().getSkillCount() > 0">
-    <ul>
-      <li ng-repeat="skill in getCollectionNode().getPrerequisiteSkillList().getBindableSkills() track by $index">
-        <em><[skill]></em>
-        <button ng-click="removePrerequisiteSkill($index)">Delete</button>
-      </li>
-    </ul>
-  </div>
-  <input ng-model="prerequisiteSkillName" type="text" placeholder="Skill name">
-  <button ng-click="addNewPrerequisiteSkill()">Add Prerequisite Skill for Exploration</button>
-  <br>
-
-  <p> Acquired skills: </p>
-  <div ng-if="getCollectionNode().getAcquiredSkillList().getSkillCount() > 0">
-    <ul>
-      <li ng-repeat="skill in getCollectionNode().getAcquiredSkillList().getBindableSkills() track by $index">
-        <em><[skill]></em>
-        <button ng-click="removeAcquiredSkill($index)">Delete</button>
-      </li>
-    </ul>
-  </div>
-  <input ng-model="acquiredSkillName" type="text" placeholder="Skill name">
-  <button ng-click="addNewAcquiredSkill()">Add Acquired Skill for Exploration</button>
-  <br>
-  <br>
   <br>
 </script>
+
+<style>
+  .glyphicon-remove-circle {
+    color: #EF5350;
+    top: -20px;
+    left: 5px;
+  }
+
+  .collection-editor-node-title {
+    padding-left: 16px;
+  }
+
+  /* TODO(mgowano): Implement horizontal scrolling using Javascript. */
+  body {
+    overflow-x: auto !important;
```

Comment:
You are right...it also works without the !important. I have removed it. However this is here temporarily.